prompts/guide_strategy1_cot.md

```diff
@@ -56,30 +100,69 @@ Is this main content (200-400 words)?
 
 **For page.plain-chunk:**
 
-1. Convert text to valid Markdown
+1. Convert text to valid Markdown following formatting rules below
 2. Create Header in Title Case
 3. **Insert image links where appropriate** (see Image Handling below)
 4. Skip Question, ConstructedResponse, KeyPhrase
 
+**Markdown Formatting Rules:**
+
+Apply these formatting rules to create a properly formatted digital textbook:
+
+1. **Paragraphs**: Separate distinct paragraphs with blank lines (`\n\n`)
+   - Do NOT add line breaks within a paragraph unless it's a natural paragraph break
+   - Each paragraph should flow continuously without internal newlines
+   
+2. **Text Formatting** (preserve from source or add as appropriate):
+   - **Bold** (`**text**`): Use for emphasis, key terms, important concepts, headers within text
+   - *Italic* (`*text*`): Use for book/journal titles, foreign words, subtle emphasis, definitions
+   - ***Bold + Italic*** (`***text***`): Use sparingly for very strong emphasis
+   
+3. **Lists** (preserve indentation):
+   - Numbered lists: `1. Item\n2. Item`
+   - Bulleted lists: `- Item\n- Item` or `* Item\n* Item`
+   - Nested lists: Use 2-space indentation for each level
+     ```
+     - Item 1
+       - Sub-item 1.1
+       - Sub-item 1.2
+     - Item 2
+     ```
+   
+4. **Blockquotes** (preserve structure):
+   - Use `> ` for quoted text or callouts
+   - Nested blockquotes: `> > Text`
+   - Maintain blank lines within blockquotes: `> \n> `
+
+5. **Inline Elements**:
+   - Code: `` `code` `` for technical terms, variable names
+   - Math: `$formula$` for inline equations
+   - Special characters: Use as-is (`&`, `—`, `–`, etc.)
+
 ### Step 5.1: Handle Images (If Provided)
 
+**CRITICAL: ALL images from metadata MUST be included. Never skip images that have captions.**
```

Comment:
This is too strong. Replace with "All images with captions (i.e. Figures) should be included in the iTELL JSON"